meetings/README.md

```diff
@@ -0,0 +1,4 @@
+As meetings are planned above, they should be entered into our
+[Google Calendar](https://calendar.google.com/calendar?cid=aXFtYmE4MmQ2MGUwdDRycGIzZ2U0MjhwbTBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+that you can subscribe to.  If you are a TSC member and need the ability to post events, send Ross your address to give permissions to.
```

Comment:
Giving people permissions to the calendar should be addressed under the [onboarding document](https://github.com/planetarysoftware/TSC/blob/master/ONBOARDING.md)

Reply:
There only, or both places?

Reply:
I would say there only, the readme should be for the general viewing public.